infra/buckets/s3.tf

```diff
@@ -3,7 +3,7 @@ data "aws_caller_identity" "current" {}
 locals {
   common_tags = "${map(
     "managedBy" , "terraform",
-    "Name"      , "${var.DSS_INFRA_TAG_PROJECT}-${var.DSS_DEPLOYMENT_STAGE}-${var.DSS_INFRA_TAG_SERVICE}",
+    "Name"      , "${var.DSS_INFRA_TAG_SERVICE}-cloudstorage",
```

Comment:
Can we specify which cloud?, e.g. something like `-s3-storage` and `-gs-storage`

Reply:
added tagging to google + differentiation of cloud